seta-react/src/layouts/CommunityLayout/CommunityLayout.tsx

```diff
@@ -31,9 +17,7 @@ const CommunityLayout = () => {
           </ScopeProvider>
 
           <Flex direction="column" align="center" sx={{ width: '100%', padding: '2rem' }}>
-            <NotificationsProvider>
-              <Outlet />
-            </NotificationsProvider>
+            <Outlet />
           </Flex>
```

Comment:
As discovered during a conversation with @pandrei02, the `<NotificationsProvider>` is still present in `<AppLayout>`, meaning it will continue to be called for ALL the pages of the app, even the ones that don't require authentication.